torch/signal/windows/windows.py

```diff
@@ -296,3 +297,83 @@ def gaussian(
                        requires_grad=requires_grad)
 
     return torch.exp(-k ** 2)
+
+
+@_add_docstr(
+    r"""
+Computes the Kaiser window
+
+The Kaiser window is defined as follows:
+
+.. math::
+    out_i = I_0 \left( \beta \sqrt{1 - \left( {\frac{i - N/2}{N/2}} \right) ^2 } \right) / I_0( \beta )
+
+Where :math:`I_0` is the zeroth order modified Bessel function of the first kind (see :func:`torch.i0`) and
+``N = M - 1`` if :attr:`sym` is True and ``M`` if :attr:`sym` is False,
```

Comment:
ug, our doc's rendering of mathematical expressions like this is horrible. I would just remove the annotations from the math and write "N = M - 1 if sym is True and M if sym is False"

Reply:
My kingdom for a half-decent sphinx theme.

Reply:
So @mruberry you don't like `:math:`I_0``, right? Personally, I think it looks kinda cool.

Reply:
@lezcano @mruberry can we put it like this? I think it's 1000% more readable than the current one:

![image](https://user-images.githubusercontent.com/13088001/197431310-afcd9e6c-870e-4c5a-83da-8acb8725f8a8.png)

For comparison, here's the current one 🤮 

![image](https://user-images.githubusercontent.com/13088001/197431468-83111ffa-2ad9-46ac-abce-2207369a15aa.png)

Reply:
I like the special link. It's also true that our rendering of double ticks in Sphinx is less than good. Now, simple ticks should be a good compromise. The thing with LaTeX is that it takes some time to render, so that's why we tend to use it just for large equations.
Also, I think that rather than just saying yhat when yhe window is symmetric something happens, I like pointing the user to the relevant arg.

Reply:
I think I prefer using double ticks. I removed the `:math:` annotation and applied back your last suggestion.